src/main/java/bio/terra/janitor/common/CredentialUtils.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.janitor.common;
+
+import bio.terra.janitor.common.exception.JanitorCredentialException;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+
+/** Utilities for handling credentials. */
+public class CredentialUtils {
+  /** Gets the {@link ServiceAccountCredentials} from a file path. */
+  public static ServiceAccountCredentials getGoogleCredentialsOrDie(String serviceAccountPath) {
+    try {
+      return ServiceAccountCredentials.fromStream(
+          Thread.currentThread().getContextClassLoader().getResourceAsStream(serviceAccountPath));
+    } catch (Exception e) {
+      throw new JanitorCredentialException(
```

Comment:
I think having a special exception class for this is overkill. This only happens initially at startup, so I'm not worried about trying to use a new class to trace it down.

Reply:
Done